app/models/weet.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const Weet = sequelize.define(
+    'Weet',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      userId: {
+        type: DataTypes.INTEGER,
+        allowNull: false
+      },
+      content: {
+        type: DataTypes.STRING,
+        allowNull: false
+      }
+    },
+    {
+      timestamps: false,
+      underscored: true,
+      tableName: 'weets-user'
+    }
+  );
```

Comment:
Since `userId` references a record in the users table you should find a way to relate the two models